datadog_checks_base/tests/test_http.py

```diff
@@ -271,6 +271,19 @@ def test_config_kerberos_keytab_file(self):
 
         assert os.environ.get('KRB5_CLIENT_KTNAME') is None
 
+    def test_config_kerberos_cache(self):
+        instance = {'kerberos_cache': '/test/file'}
+        init_config = {}
+
+        http = RequestsWrapper(instance, init_config)
+
+        assert os.environ.get('KRB5CCNAME') is None
+
+        with mock.patch('requests.get', side_effect=lambda *args, **kwargs: os.environ.get('KRB5CCNAME')):
+            assert http.get('https://www.google.com') == '/test/file'
+
+        assert os.environ.get('KRB5CCNAME') is None
```

Comment:
Could you add a test case with envvar `KRB5CCNAME` already populated to test that the original value is restored ?